dast/__main__.py

```diff
@@ -37,9 +39,8 @@ def main(then_path: str, now_path: str, verbose: bool = True):
 
     diff = DeepDiff(then_ast, now_ast)
     if diff and verbose:
+        print(f"diff for file {now_path}")
         print(diff)
 
-    return now_ast, then_ast
-
 if __name__ == "__main__":
-    now, then = main(sys.argv[1], sys.argv[2])
```

Comment:
These are quite nice for debugging. If you run the script as `python -im dast a.py b.py` you can then inspect the ASTs in interact mode

Reply:
a9bff52